convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -979,35 +963,86 @@ def test_check_installed_rhel_kernel_returns_false(self):
         self.assertEqual(pkghandler.is_rhel_kernel_installed(), False)
 
     @unit_tests.mock(pkghandler, "get_third_party_pkgs", lambda: [])
-    @unit_tests.mock(logger.CustomLogger, "info", LogMocked())
+    @unit_tests.mock(pkghandler.logging, "getLogger", GetLoggerMocked())
     def test_list_third_party_pkgs_no_pkgs(self):
         pkghandler.list_third_party_pkgs()
 
-        self.assertEqual(logger.CustomLogger.info.msg, "No third party packages installed.\n")
+        self.assertTrue("No third party packages installed" in pkghandler.logging.getLogger.info_msgs[0])
 
     @unit_tests.mock(pkghandler, "get_third_party_pkgs", GetInstalledPkgsWFingerprintsMocked())
     @unit_tests.mock(pkghandler, "print_pkg_info", PrintPkgInfoMocked())
-    @unit_tests.mock(logger.CustomLogger, "warning", LogMocked())
+    @unit_tests.mock(pkghandler.logging, "getLogger",  GetLoggerMocked())
     @unit_tests.mock(utils, "ask_to_continue", DumbCallableObject())
     def test_list_third_party_pkgs(self):
         pkghandler.list_third_party_pkgs()
 
         self.assertEqual(len(pkghandler.print_pkg_info.pkgs), 3)
-        self.assertTrue("Only packages signed by" in logger.CustomLogger.warning.msg)
+        self.assertTrue("Only packages signed by" in pkghandler.logging.getLogger.warning_msgs[0])
 
     @unit_tests.mock(tool_opts, "disablerepo", ['*', 'rhel-7-extras-rpm'])
     @unit_tests.mock(tool_opts, "enablerepo", ['rhel-7-extras-rpm'])
-    @unit_tests.mock(logger.CustomLogger, "warning", LogMocked())
+    @unit_tests.mock(pkghandler.logging, "getLogger", GetLoggerMocked())
     def test_is_disable_and_enable_repos_has_same_repo(self):
         pkghandler.has_duplicate_repos_across_disablerepo_enablerepo_options()
-        self.assertTrue("Duplicate repositories were found" in logger.CustomLogger.warning.msg)
+        self.assertTrue("Duplicate repositories were found" in pkghandler.logging.getLogger.warning_msgs[0])
 
     @unit_tests.mock(tool_opts, "disablerepo", ['*'])
     @unit_tests.mock(tool_opts, "enablerepo", ['rhel-7-extras-rpm'])
-    @unit_tests.mock(logger.CustomLogger, "warning", LogMocked())
+    @unit_tests.mock(pkghandler.logging, "getLogger", GetLoggerMocked())
     def test_is_disable_and_enable_repos_doesnt_thas_same_repo(self):
         pkghandler.has_duplicate_repos_across_disablerepo_enablerepo_options()
-        self.assertEqual(logger.CustomLogger.warning.called, 0)
+        self.assertEqual(len(pkghandler.logging.getLogger.warning_msgs), 0)
+
+    @unit_tests.mock(system_info, "name", "Oracle Linux Server release 7.9")
+    @unit_tests.mock(system_info, "arch", "x86_64")
+    @unit_tests.mock(system_info, "version", namedtuple("Version", ["major", "minor"])(7, 9))
+    @unit_tests.mock(pkghandler.logging, "getLogger", GetLoggerMocked())
+    @unit_tests.mock(utils, "get_file_content", lambda x: "UPDATEDEFAULT=yes\nDEFAULTKERNEL=kernel-uek\n")
```

Comment:
Nit: here and a couple of other places, use `lambda _: ...`, this says that this arg is not used

Reply:
Thanks... applied in my last commit